src/foam/nanos/auth/PublicUserEntity.js

```diff
@@ -0,0 +1,73 @@
+foam.CLASS({
+  package: 'foam.nanos.auth',
+  name: 'PublicUserEntity',
+  documentation: `This model represents a public subset of a user's properties`,
+
+  javaImports: ['foam.nanos.auth.User'],
+
+  properties: [
+    {
+      class: 'Long',
+      name: 'id',
+      visibility: foam.u2.Visibility.RO
+    },
+    {
+      class: 'String',
+      name: 'firstName',
+      visibility: foam.u2.Visibility.RO
+    },
+    {
+      class: 'String',
+      name: 'lastName',
+      visibility: foam.u2.Visibility.RO
+    },
+    {
+      class: 'EMail',
+      name: 'email'
+    },
+    {
+      class: 'foam.nanos.fs.FileProperty',
+      name: 'profilePicture',
+      view: { class: 'foam.nanos.auth.ProfilePictureView' }
+    }
+  ],
+  axioms: [
+    {
+      writeToSwiftClass: function(cls) {
+        cls.method(foam.swift.Method.create({
```

Comment:
Why do we use objects for swift and strings for java?

Reply:
@shreymahendru can answer this, all I did was rename this file.

Reply:
`writeToSwiftClass` should be removed altogether. Cause there is no need for this method on the client-side as an Entity object is not supposed to be initialized there. I figured we would need it for the deserialization of the DTO.